### ISSUE TITLE
Cannot find type definition file for 'src/vue-shims'

### ISSUE THREAD

**Issue by cexbrayat** [vuejs/vue-test-utils-next#61]
Thank you for releasing this alpha so we can give it a spin!

Using `alpha.1` in a TypeScript project throws a compilation error:

```
ERROR  Failed to compile with 2 errors                                                                                              14:58:14

error  in /Users/ced-pro/Code/vtu-next-test/node_modules/@vue/test-utils/dist/components/RouterLinkStub.d.ts

ERROR in /Users/ced-pro/Code/vtu-next-test/node_modules/@vue/test-utils/dist/components/RouterLinkStub.d.ts(1,23):
1:23 Cannot find type definition file for 'src/vue-shims'.
 > 1 | /// <reference types="src/vue-shims" />
     |                       ^
   2 | export declare const RouterLinkStub: new () => import("vue").ComponentPublicInstance<{
   3 |     to: any;
   4 | } & {}, unknown, unknown, {}, {}, Record<string, any>, import("vue").VNodeProps & {

error  in /Users/ced-pro/Code/vtu-next-test/node_modules/@vue/test-utils/dist/mount.d.ts

ERROR in /Users/ced-pro/Code/vtu-next-test/node_modules/@vue/test-utils/dist/mount.d.ts(9,9):
9:9 Property 'default' of type 'string | VNode<RendererNode, RendererElement> | { render: Function; } | undefined' is not assignable to string index type 'Slot'.
    7 |     props?: Record<string, any>;
    8 |     slots?: {
 >  9 |         default?: Slot;
      |         ^
   10 |         [key: string]: Slot;
   11 |     };
   12 |     global?: {
```

This can be easily reproduced in @lmiller1990 demo repo (even if the Vue CLI setup reports a more detailled error):

```
git clone https://github.com/lmiller1990/vtu-next-demo.git
cd vtu-next-demo
yarn
yarn tsc
```

**Comment by lmiller1990** [comment]
Yep, providing better tooling to compile the tests into JS is something we should do before the beta for sure. Good catch - I've just been running `yarn jest` to test things out, but for a large test suite transpiling to JS will definitely speed things up, I didn't think this far ahead.

Running `yarn jest` should at least yield passing tests if you want to play around.

**Comment by lmiller1990** [comment]
Related, #143 maybe. Same log with RouterStub.

**Comment by cexbrayat** [comment]
Looks like all issues are now fixed in VTU 2.0.0-beta.0, allowing to remove `skipLibCheck` for those who want to.
